Diffusion/Pipeline/PipelineLoader.swift

```diff
@@ -82,20 +90,20 @@ extension PipelineLoader {
         return url.lastPathComponent
     }
     
-    var downloadedPath: Path { PipelineLoader.models / filename }
-    var downloadedURL: URL { downloadedPath.url }
+    var downloadedURL: URL { PipelineLoader.models.appendingPathComponent(filename) }
 
-    var uncompressPath: Path { downloadedPath.parent }
+    var uncompressURL: URL { PipelineLoader.models.deletingLastPathComponent() }
```

Comment:
```suggestion
    var uncompressURL: URL { downloadedURL.deletingLastPathComponent() }
```

This must be inside the models folder. `PipelineLoader.models` should also work, I think.

Reply:
Going with PipelineLoader.models.